src/app/runs/[id]/_Tabs/Configuration/DockerImageCollapsible.tsx

```diff
@@ -0,0 +1,85 @@
+import ChevronDown from "@/assets/icons/chevron-down.svg?react";
+import {
+	CollapsibleContent,
+	CollapsibleHeader,
+	CollapsiblePanel,
+	CollapsibleTrigger,
+	Tag
+} from "@zenml-io/react-component-library";
+import { useState } from "react";
+import Docker from "@/assets/icons/docker.svg?react";
+import Redirect from "@/assets/icons/redirect.svg?react";
+import { CopyButton } from "@/components/CopyButton";
+import { Codesnippet } from "@/components/CodeSnippet";
+import { KeyValue } from "@/components/KeyValue";
+
+type Props = {
+	data: {
+		image: string;
+		contains_code: boolean;
+		dockerfile: string;
+		requirements: string;
+	};
+};
+
+export function DockerImageCollapsible({ data }: Props) {
+	const [open, setOpen] = useState(true);
+
+	return (
+		<CollapsiblePanel open={open} onOpenChange={setOpen}>
+			<CollapsibleHeader intent="primary" className="flex items-center gap-[10px]">
+				<CollapsibleTrigger className="flex w-full items-center gap-[10px]">
+					<ChevronDown
+						className={` ${
+							open ? "" : "-rotate-90"
+						} h-5 w-5 rounded-md fill-neutral-500 transition-transform duration-200 hover:bg-neutral-200`}
+					/>
+					Docker Image
+				</CollapsibleTrigger>
+			</CollapsibleHeader>
+			<CollapsibleContent className="border-t border-theme-border-moderate bg-theme-surface-primary px-5 py-3">
+				<dl className="grid grid-cols-1 gap-x-[10px] gap-y-2 md:grid-cols-3 md:gap-y-4">
+					<KeyValue
+						label="Docker Image"
+						value={
+							<div className="flex justify-between">
+								<Tag className="inline-flex items-center gap-0.5" rounded={false} emphasis="subtle">
+									<Docker className="mr-1 h-4 w-4 fill-theme-text-brand" />
+									{"zenml/my_pipeline:data"}
+								</Tag>
+								<div className="align-center mr-1 flex">
+									<a
```

Comment:
Is it correct, that it's an anchor Tag with data.image? On a first glance it would semantically make sense to have this as an image 😅